examples/lighting-app/bouffalolab/bl602/src/AppTask.cpp

```diff
@@ -168,8 +168,8 @@ void AppTask::AppTaskMain(void * pvParameter)
     CHIP_ERROR err;
 
     log_info("App Task entered\r\n");
-    log_async_init();
-    enable_async_log();
+    // log_async_init();
```

Comment:
Any reason to keep the commented out code here? Please either delete or if this is temporary, add a comment as to why, like a `TODO` that explains why it was disabled and under what conditions it should be added back. Even then it may make sense to just have the `TODO: enable async log here once ...` without the commented out code.

Reply:
async log had been dropped from master. 
These two are already deleted from this PR.